flax/nn/attention.py

```diff
@@ -197,28 +197,31 @@ def _init(shape_fn):
 class MultiHeadDotProductAttention(base.Module):
   """Multi-head dot-product attention."""
 
-  def apply(self,
-            inputs_q,
-            inputs_kv,
-            num_heads,
-            dtype=jnp.float32,
-            qkv_features=None,
-            out_features=None,
-            attention_axis=None,
-            causal_mask=False,
-            padding_mask=None,
-            key_padding_mask=None,
-            segmentation=None,
-            key_segmentation=None,
-            cache=None,
-            broadcast_dropout=True,
-            dropout_rng=None,
-            dropout_rate=0.,
-            deterministic=False,
-            precision=None,
-            kernel_init=default_kernel_init,
-            bias_init=initializers.zeros,
-            bias=True):
+  def apply(
+      self,
+      inputs_q,
+      inputs_kv,
+      num_heads,
+      dtype=jnp.float32,
+      qkv_features=None,
+      out_features=None,
+      attention_axis=None,
+      causal_mask=False,
+      padding_mask=None,
+      key_padding_mask=None,
+      segmentation=None,
+      key_segmentation=None,
+      cache=None,
+      broadcast_dropout=True,
+      dropout_rng=None,
+      dropout_rate=0.,
+      deterministic=False,
+      precision=None,
+      kernel_init=default_kernel_init,
+      bias_init=initializers.zeros,
+      bias=True,
+      attention_fn=dot_product_attention,
+  ):
```

Comment:
nit: could you reformat this to align with self in `apply(self,` put the `):` at the end of the last line, w. no trailing comma.